code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,320 @@
+/*
+Destabilized extracts:
+	Throwable extracts that apply an effect on hit.
+*/
+
+/obj/item/slimecross/destabilized
+	name = "destabilized extract"
+	desc = "It seems ready to burst at any moment."
+	effect = "destabilized"
+	icon_state = "destabilized"
+
+/obj/item/slimecross/destabilized/throw_impact(atom/A, var/datum/thrownthing/T)
+	if(!..()) //For those tactical catches
+		if(hitEffect(A, T.thrower))
+			playsound(get_turf(A), 'sound/misc/splort.ogg', get_volume_by_throwforce_and_or_w_class(), 1, -1)
+			qdel(src)
+
+/obj/item/slimecross/destabilized/proc/hitEffect(atom/A, mob/thrower)
+	return TRUE //But nothing happens...
+
+/obj/item/slimecross/destabilized/grey
+	colour = "grey"
+
+/obj/item/slimecross/destabilized/grey/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/mob/living/simple_animal/slime/random/S = new (get_turf(A))
```

Comment:
random slimes would normally require rainbow slimes in the first place, should probably be a grey slime instead

Reply:
Fair enough